recipe/meta.yaml

```diff
@@ -1,15 +1,16 @@
+{% set name = "fiona" %}
 {% set version = "1.9.5" %}
 
 package:
   name: fiona
   version: {{ version }}
 
 source:
-  url: https://github.com/Toblerity/Fiona/archive/refs/tags/{{ version }}.zip
-  sha256: 31aef8eb0402c7dcd822ffe576fccbb7bfbe29a31f3b6805b87770d4b6f266e7
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fiona-{{ version }}.tar.gz
+  sha256: 99e2604332caa7692855c2ae6ed91e1fffdf9b59449aa8032dd18e070e59a2f7
```

Comment:
This change is following up on:
https://github.com/conda-forge/fiona-feedstock/pull/219#issuecomment-1812449337